### ISSUE TITLE
Is JitsiMeetSDK only support P2P instead of using XMPP Websocket?

### ISSUE THREAD

**Issue by Symous** [jitsi/jitsi-meet#14189]
### Description:
Hi, team, I am new to Jitsi. sorry first if I raise a stupid question. I use JitsiMeetSDK in my app and build a docker jitsi server, I have just found out that when there is more than 2 users (such as 2 mobile users and 1 pc users), the video on the app can not be seen, but the PC works fine to see all the users, and when the PC leave the meet the app works fine then.

After some tests I found out that setup ENABLE_XMPP_WEBSOCKET=0 in the .env can solve this problem, but for some reasons XMPP WebSocket is important to me and I do not want to turn it off.

I guess when there in 2 users JitsiMeetSDK use P2P mode and switch to XMPP or something else that the SDK do not support so the video can not be ssen.  My question is that is JitsiMeetSDK support XMPP now and if there is any thing else I need to configure througn the SDK?  thanks!

**Comment by saghul** [comment]
That means your server is not configured correctly. A litlely reason is port 10000 is closed and thus traffic through the JVB is broken.

**Comment by Symous** [comment]
@saghul I have confirmed that the 10000 is opened and no firewall rule blocked that.

**Comment by saghul** [comment]
Please share your JS console logs. Test with 3 browsers, it's easier to gather logs.

**Comment by Symous** [comment]
@saghul are you mean thhe js console logs in browser? my problem is that the browser works well (can see all the participants with web and app) but the mobile app not(can't see others when there are more than 2 participants). so I guess the problem is related to the mobile sdk or docker server. :(

**Comment by saghul** [comment]
Is everyone in the same LAN? IS the mobile user on a different network?

**Comment by Symous** [comment]
@saghul  yes, same LAN, the laptops(Web user) and the mobile phones(App user) connect to the same Wi-Fi.

**Comment by saghul** [comment]
That is odd. What versions of the Docker images and of the SDK are you using?
